backend/cfg/cfg_equivalence.ml

```diff
@@ -204,6 +204,9 @@ let check_operation : location -> Cfg.operation -> Cfg.operation -> unit =
     ()
   | Const_float expected, Const_float result when Int64.equal expected result ->
     ()
+  | Const_float32 expected, Const_float32 result
```

Comment:
(As showed by https://github.com/ocaml-flambda/flambda-backend/pull/2719, this should not have any effects.)